src/Sylius/Bundle/CoreBundle/Controller/ExceptionController.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Controller;
+
+use FOS\RestBundle\Controller\ExceptionController as BaseExceptionController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Log\DebugLoggerInterface;
+
+/**
+ * @author Kamil Kokot <kamil.kokot@lakion.com>
+ */
+class ExceptionController extends BaseExceptionController
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function showAction(Request $request, $exception, DebugLoggerInterface $logger = null)
+    {
+        if ($this->container->hasParameter('sylius.show_exception') && $this->container->getParameter('sylius.show_exception')) {
```

Comment:
Instead of calling this always, maybe add listener in test env that will be run only there, not also in production etc...